src/runtime_src/core/pcie/common/dmatest.h

```diff
@@ -80,74 +89,73 @@ namespace xcldev {
             }
 
             int result = 0;
-            for_each(threads.begin(), threads.end(), [&](std::future<int> &v) {result += v.get();});
+            std::for_each(threads.begin(), threads.end(), [&](std::future<int> &v) {result += v.get();});
+            return result;
+        }
+
+        void clear() {
+            //Clear out the host shadow buffer
+            std::for_each(mBOList.begin(), mBOList.end(), [&](std::pair<xclBufferHandle, xrt_core::aligned_ptr_type> &v) {
+                    std::memset(v.second.get(), 'x', mSize);
```

Comment:
Strictly formatting.  If the lambda doesn't fit on the same line as the std::algorithm, then I really like it like this:
```
std::for_each(mBOList.begin(), mBOList.end(), 
              [&](std::pair<xclBufferHandle, xrt_core::aligned_ptr_type> &v) {
                    std::memset(v.second.get(), 'x', mSize);
              });
```
That way it looks and feels a lot more like a regular function.